.github/workflows/main.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - 'master'
+    - 'dev'
```

Comment:
Why is that necessary? We build pull requests already, isn't that enough?

Reply:
On the projects in my school, we are working on the dev branch, we merge into the dev branch, once everything is validated and the tests work, we merge with the branch master, can I remove if you want ?

Reply:
yeah I think it does not need to be in the PR :+1:

Reply:
It is deleted from CI ^^